swatch-producer-aws/src/main/java/com/redhat/swatch/processors/BillableUsageProcessor.java

```diff
@@ -92,7 +91,8 @@ public void process(BillableUsage billableUsage) {
       MDC.put("account_id", billableUsage.getAccountNumber());
     }
 
-    if (!isApplicable(billableUsage)) {
+    Optional<Metric> metric = lookupMetric(billableUsage);
```

Comment:
nitpick: method name should reflect that this also rejects non-AWS usage. Maybe `validateUsageAndLookupMetric` ? Alternatively, we could keep them separated. Something like:

```java
var isValid = isApplicable(billableUsage);
Optional<Metric> metric = Optional.empty();
if (isValid) {
  metric =  lookupMetric(billableUsage);
}
if (!isValid || metric.isEmpty()) {
```
Either would be fine to me

Reply:
Method name updated to `validateUsageAndLookupMetric`